data/json/recipes/recipe_food.json

```diff
@@ -5894,6 +5894,23 @@
       ]
     ]
   },
+  {
+    "type": "recipe",
+    "result": "sausage_wasteland",
```

Comment:
You now have two recipes with same result, so you need to add suffix to one or both of the recipes. Otherwise only one recipe would be available to characters.

Reply:
```suggestion
    "result": "sausage_wasteland",
    "id_suffix": "smoke",
```
And we shouldn't add suffix to the old recipe. That would probably cause problems with old saves.